### ISSUE TITLE
Active heading level's subscript color

### ISSUE THREAD

**Issue by spencerbaynton** [WordPress/gutenberg#5391]
## Issue Overview

There is currently no CSS rule for styling the subscript of the heading level when its button `is-active`. The subscript appears invisible because its color is the same as the background color of the button.

## Steps to Reproduce

1. Create a new heading block.
2. Note that in the toolbar the **2** subscript is not visible in the active **H<sub>2</sub>** button.

Chrome 64.0.3282.186
Firefox 58.0.2
Gutenberg 2.3.0

## Expected Behavior

The heading level subscript should be `$white` when the heading level's button `is-active`.

## Current Behavior

There is no styling for the subscript when the heading level's button `is-active`.

## Possible Solution

The button's active styling exists in [components/toolbar/style.scss#L91](https://github.com/WordPress/gutenberg/blob/e8d13377e5542440ec321dc03a093e283a1dc499/components/toolbar/style.scss#L91). This might be the appropriate location to add the subscript's active styling as well. Something like this might be all that is necessary:

```scss
.components-toolbar__control.components-button {
	...
	&:not(:disabled).is-active:after {
		color: $white;
	}
	...
}
```

## Screenshots / Video

![screen shot 2018-03-04 at 15 12 42](https://user-images.githubusercontent.com/1483584/36950067-c6644d74-1fbe-11e8-9f55-7f89a62d28f9.png)

Also, in the block's settings.

![screen shot 2018-03-04 at 15 20 20](https://user-images.githubusercontent.com/1483584/36950142-a0ad7cf8-1fbf-11e8-874c-3f93299ea8f4.png)

## Todos

- [ ] Add new CSS rule

**Comment by amdrew** [comment]
This has now been fixed in v2.4: https://github.com/WordPress/gutenberg/pull/5560

**Comment by jorgefilipecosta** [comment]
> This has now been fixed in v2.4: #5560

Yes, that's true, closing the issue.
